planner/core/explain.go

```diff
@@ -238,7 +238,14 @@ func (p *PhysicalHashJoin) ExplainInfo() string {
 	}
 
 	buffer.WriteString(p.JoinType.String())
-	fmt.Fprintf(buffer, ", inner:%s", p.Children()[p.InnerChildIdx].ExplainID())
+	var InnerChildIdx = p.InnerChildIdx
+	if p.UseOuterToBuild && ((p.JoinType == LeftOuterJoin && InnerChildIdx == 1) || (p.JoinType == RightOuterJoin && InnerChildIdx == 0)) {
+		InnerChildIdx = 1 - InnerChildIdx
+	}
+	fmt.Fprintf(buffer, ", inner:%s", p.Children()[InnerChildIdx].ExplainID())
+	if p.UseOuterToBuild {
+		buffer.WriteString(" (REVERSED)")
```

Comment:
Here keep it, but it will updated in the issue #12985

Reply:
Add the issue link as a TODO in the code.

Reply:
ok